src/GeoPing.Core/Services/IAccountService.cs

```diff
@@ -19,6 +19,7 @@ public interface IAccountService
         Task<OperationResult> ConfirmResetAsync(string userId, string token, string newPassword);
         Task ConfirmAccountWithoutEmailAsync(string userEmail);
         OperationResult GetProfile(Guid gpUserId);
+        OperationResult GetShortProfile(Guid userId);
```

Comment:
What data type inside the Data property of this result?

Reply:
Corrected in new commit